### PR TITLE
WIP [kesch] Updated software list for Kesch with RH7.5

### PR DESCRIPTION
**Pull request by lucamar**
The pull request includes the recipe to build `nmon` (`BinariesTarball`).

### CODE REVIEW

File: easybuild/easyconfigs/c/CMake/CMake-3.13.0.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+sanity_check_paths = {
```
Comment (victorusu) [inline easybuild/easyconfigs/c/CMake/CMake-3.13.0.eb]: Does it make sense to add the original `osdependencies`?
```
osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel'), 'ncurses-devel']
```

File: easybuild/easyconfigs/h/HDF5/HDF5-1.10.4-PGI-18.10-GCC-5.4.0-2.26-serial.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# @author: Guilherme Peretti Pezzi and Luca Marsella (CSCS)
+
+easyblock = 'ConfigureMake'
+
+name = 'HDF5'
+version = '1.10.4'
+versionsuffix = '-serial'
```
Comment (victorusu) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.10.4-PGI-18.10-GCC-5.4.0-2.26-serial.eb]: Ins't the HDF5 serial and the parallel version called HDF5-parallel?

File: jenkins-builds/MCH-RH7.4-17.02 [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ NCO-4.6.2-gmvolf-17.02.eb
+ nmon-16g.eb                                         --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ PrgEnv-gnu-kesch-gmvolf-17.02.eb                    --module-naming-scheme=EasyBuildMNS
+ gmvapich2-17.02_cuda_8.0_gdr.eb
+ matlab-r2016a.eb                                    --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ Python-2.7.13-gmvolf-17.02.eb
+ Python-3.6.2-gmvolf-17.02.eb
+ R-3.3.2-gmvolf-17.02-libX11-1.6.3.eb
+ R-3.5.2-gmvolf-17.02-libX11-1.6.3.eb
+ reframe-2.14.eb                                     --set-default-module
```
Comment (victorusu) [inline jenkins-builds/MCH-RH7.4-17.02]: Small thing, but reframe is already 2.15. You could update it here or we could make a PR later to add the 2.15 version.

File: jenkins-builds/MCH-RH7.4-17.02 [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ Ruby-2.5.3-gmvolf-17.02.eb
+ ncview-2.1.7-gmvolf-17.02.eb
+ netCDF-C++-4.3.0-gmvolf-17.02.eb
+ netCDF-Fortran-4.4.4-gmvolf-17.02.eb
+ netcdf4-python-1.3.0-gmvolf-17.02-Python-2.7.13.eb
+ netcdf4-python-1.3.0-gmvolf-17.02-Python-3.6.2.eb
+ netCDF-4.6.1-PGI-18.10-GCC-5.4.0-2.26.eb            --installpath=/apps/escha/UES/jenkins/RH7.4/pgi_PE18.10/easybuild 
+ netCDF-C++-4.3.0-PGI-18.10-GCC-5.4.0-2.26.eb        --installpath=/apps/escha/UES/jenkins/RH7.4/pgi_PE18.10/easybuild
+ netCDF-Fortran-4.4.4-PGI-18.10-GCC-5.4.0-2.26.eb    --installpath=/apps/escha/UES/jenkins/RH7.4/pgi_PE18.10/easybuild
+ PrgEnv-pgi-kesch-18.10.eb                           --installpath=/apps/escha/UES/jenkins/RH7.4/pgi_PE18.10/easybuild --module-naming-scheme=EasyBuildMNS
```
Comment (victorusu) [inline jenkins-builds/MCH-RH7.4-17.02]: Is it possible to sort the file? It is a bit though to identify the installed packages.

Comment (lucamar) [reply jenkins-builds/MCH-RH7.4-17.02]: @victorusu I kept the original ordering, we can certainly re-arrange the items though.

File: jenkins-builds/MCH-RH7.4-17.02 (hunk 3 above)
Comment (victorusu) [reply jenkins-builds/MCH-RH7.4-17.02]: Actually now ReFrame is 2.16... 😁

File: easybuild/easyconfigs/h/HDF5/HDF5-1.10.4-PGI-18.10-GCC-5.4.0-2.26-serial.eb (hunk 2 above)
Comment (lucamar) [reply easybuild/easyconfigs/h/HDF5/HDF5-1.10.4-PGI-18.10-GCC-5.4.0-2.26-serial.eb]: That's the naming convention used previously for `Kesch`.

**Comment by lucamar** [comment]
I'm closing this pull request as we have changed the branch from wich we would like to merge (`mch7.5`).

**Pull request closed without merging**